rs/types/management_canister_types/tests/ic.did

```diff
@@ -354,6 +354,20 @@ type http_request_args = record {
     pricing_version: opt nat32;
 };
 
+type flexible_http_request_args = record {
+    url : text;
+    max_response_bytes : opt nat64;
+    headers : vec http_header;
+    method : variant { get; head; post };
+    body : opt blob;
+    transform : opt record {
+        function : func (record {response : http_response; context : blob}) -> (http_response) query;
+        context : blob;
+    };
+    is_replicated : opt bool;
+    pricing_version : opt nat32;
```

Comment:
Why do we need a pricing version and `is_replicated` here? Also, do we need the return type?